pallets/circuit-breaker/src/tests/mock.rs

```diff
@@ -86,6 +86,7 @@ frame_support::construct_runtime!(
 		Omnipool: pallet_omnipool,
 		Tokens: orml_tokens,
 		CircuitBreaker: pallet_circuit_breaker,
+		AmmSupport: pallet_amm_support,
```

Comment:
This should probably be renamed to just support or swap or chain support or smth as it's not just used by amms

Reply:
What did we decide?